frontend/simple/views/containers/sidebar/Activity.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+    <menu-parent class="level-right">
+      <menu-trigger class="is-icon">
+        <i class="fa" :class="activityCount ? 'fa-bell' : 'fa-bell-o'"></i>
+        <span class="c-badge" v-if="activityCount" data-test="alertNotification">{{ activityCount }}</span>
+      </menu-trigger>
+
+      <menu-content class="c-content">
+        <menu-header>
+          <i18n>Notifications</i18n>
+        </menu-header>
+        <list hasMargin>
```

Comment:
How does this component receive / know about `hasMargin`?

Reply:
Oh, nevermind, it is creating a list and is telling it that it has the margin... I see.

Reply:
I'm not sure I like this. This is confusing. Why do it this vue-specific way when normal CSS classes work just fine?

Reply:
General rule of thumb: if there are two different ways to do something, and both do the same thing, and one is simpler than the other, do it the simpler way.

Using this vue.js feature when it's not needed and all it does is apply a class is using CSS + Vue.js feature, and unclear why the latter is necessary.

It would work just the same with just pure CSS, right? Then that is the simpler and more correct solution.

Reply:
When tomorrow we have 3 places with the same css rule, we'll have `margin: 1rem 0` duplicated 3 times. So this prop prevents that from happening.

_Still..._, what I've done is what it's know as _wrong abstraction_ predicting the future. And human beings are horrible predicting the future. So, I'll leave it as raw css and when the duplication comes in (if it does) we abstract it.
Until then, _Prefer duplication over the wrong abstraction_

Reply:
Actually, we already have 2 places where this `prop` is being used (and not only one as I thought), so I think the prop is okay here.
If we use only a class, we would have to write

```scss
.gi-list {
  margin: $gi-spacer-sm 0;
}
```

twice in our code... With a prop we avoid that.

Reply:
Why? Just put it into the global scss?

Reply:
IMO I don't consider this a global class, when it's specific to a component only.

And I would like to avoid adding global classes all the time, otherwise in a near future our "global CSS scope" would be full of trash and probably lost classes.